test/test_mbr.ml

```diff
@@ -81,17 +81,17 @@ let test_make_too_many_partitions () =
   | Error _ -> ()
 
 let test_make_overlapping () =
-  let p1 = get_ok (Mbr.Partition.make 10l 10l) in
-  let p2 = get_ok (Mbr.Partition.make 15l 10l) in
-  match (Mbr.make [ p1; p2 ], Mbr.make [ p2; p1 ]) with
+  let p1 = get_ok (Mbr.Partition.make ~partition_type:6 10l 10l) in
+  let p2 = get_ok (Mbr.Partition.make ~partition_type:6 15l 10l) in
+  match (Mbr.make 6 [ p1; p2 ], Mbr.make 6 [ p2; p1 ]) with
```

Comment:
We have not changed `Mbr.make`. The partition type is only relevant for the partition constructor.
```suggestion
  match (Mbr.make [ p1; p2 ], Mbr.make [ p2; p1 ]) with
```

Reply:
I see, sorry for overlooking this!